### DISCUSSION TITLE
When would you use MiniRocketPlus vs MiniRocket?

### DISCUSSION THREAD

**Discussion by hihosilvers** [timeseriesAI/tsai#758] [Q&A] [answered]
Is MiniRocketPlus just giving more hyperparameter control over the kernels and convolutions or is there something else?

MiniRocketPlus and MiniRocket should give the same shape of outputs given the same `num_features` right?

Also, when would you use InceptionRocket? What is the relationship (if any) between InceptionTime and InceptionRocket?

Thank you!

**Comment by oguiza** [comment, marked as answer]
Hi @teejK,
In general, I recommend always using the Plus version of the models if it's available. Normal versions try to reproduce the models in the original paper, while the Plus version is an adaptation that allows you to use more hyperparameters and it also automatically selects a custom_head to recreate an output of the same shape as the input.

**Reply by Arderiu** [reply]
It is still note clear to me what is the relationship (if any) between InceptionTime and InceptionRocket?
